common/views/components/ImageViewer/ImageViewer.js

```diff
@@ -68,207 +84,112 @@ const ImageWrapper = styled.div`
   }
 `;
 
-function getTileSources(data) {
-  return [
-    {
-      '@context': 'http://iiif.io/api/image/2/context.json',
-      '@id': data['@id'],
-      height: data.height,
-      width: data.width,
-      profile: ['http://iiif.io/api/image/2/level2.json'],
-      protocol: 'http://iiif.io/api/image',
-      tiles: [
-        {
-          scaleFactors: [1, 2, 4, 8, 16, 32],
-          width: 400,
-        },
-      ],
-    },
-  ];
-}
 type ImageViewerProps = {|
   id: string,
-  src: string,
-  srcSet: string,
   width: number,
   height?: number,
   infoUrl: string,
   lang: ?string,
+  alt: string,
   tabbableControls: boolean,
+  urlTemplate: IiifUriOpts => string,
+  presentationOnly?: boolean,
```

Comment:
[We never seem to pass this prop through](https://github.com/wellcometrust/wellcomecollection.org/pull/4906/files#diff-73c6a0583e43bee87b93420f87e90ce6R57-R65) - so where are we using it?

Reply:
https://github.com/wellcometrust/wellcomecollection.org/blob/035fb847022ddefcd702800a5d5b50ce3d960e4a/common/views/components/IIIFViewer/IIIFViewer.js#L803